bundles/org.openhab.binding.solarmax/README.md

```diff
@@ -0,0 +1,64 @@
+| acPhase1Current          | Number:ElectricCurrent   | Ac Phase 1 Current in Amps                   |
+| acPhase2Current          | Number:ElectricCurrent   | Ac Phase 2 Current in Amps                   |
+| acPhase3Current          | Number:ElectricCurrent   | Ac Phase 3 Current in Amps                   |
+| energyGeneratedToday     | Number:Energy            | Energy Generated Today in Wh                 |
+| energyGeneratedTotal     | Number:Energy            | Energy Generated since recording began in Wh |
+| operatingHours           | Number                   | Operating Hours since recording began in h   |
+| energyGeneratedYesterday | Number:Energy            | Energy Generated Yesterday in Wh             |
+| energyGeneratedLastMonth | Number:Energy            | Energy Generated Last Month in Wh            |
+| energyGeneratedLastYear  | Number:Energy            | Energy Generated Last Year in Wh             |
+| energyGeneratedThisMonth | Number:Energy            | Energy Generated This Month in Wh            |
+| energyGeneratedThisYear  | Number:Energy            | Energy Generated This Year in Wh             |
+| currentPowerGenerated    | Number:Power             | Power currently being generated in W         |
+| acFrequency              | Number:Frequency         | AcFrequency in Hz                            |
+| acPhase1Voltage          | Number:ElectricPotential | Ac Phase1 Voltage in V                       |
+| acPhase2Voltage          | Number:ElectricPotential | Ac Phase2 Voltage in V                       |
+| acPhase3Voltage          | Number:ElectricPotential | Ac Phase3 Voltage in V                       |
+| heatSinkTemperature      | Number:Temperature       | Heat Sink Temperature in degrees celcius     |
+
+### SolarMax Commands
+
+During the implementation the SolarMax device was sent all possible 3 character commands and a number of 4 character commands, to see what it responded to.
+The most interesting, identifiable and useful commands were implemented as channels above.
+
+Here is a list of other commands, which are know to return some kind of value: ADR (DeviceAddress / Device Number - only used if the devices are linked serially), AMM, CID, CPG, CPL, CP1, CP2, CP3, CP4, CP5, CYC, DIN, DMO, ETH, FH2, FQR, FWV, IAA, IED, IEE, IEM, ILM, IP4, ISL, ITS, KFS, KHS, KTS, LAN (Language), MAC (MAC Address), PAE, PAM, PDA, PDC, PFA, PIN (Power Installed), PLR, PPC, PRL (AC Power Percent, PSF, PSR, PSS, QAC, QMO, QUC, RA1, RA2, RB1, RB2, REL, RH1, RH2, RPR, RSD, SAC, SAL, SAM, SCH, SNM (IP Broadcast Address??), SPS, SRD, SRS, SYS (Operating State), TCP (probably port number - 12345), TI1, TL1, TL3, TND, TNH, TNL, TP1, TP2, TP3, TV0, TV1, TYP (Type?), UA2, UB2, UGD, UI1, UI2, UI3, ULH, ULL, UMX, UM1, UM2, UM3, UPD, UZK, VCM
+
+Valid commands which returned a null/empty value during testing: FFK, FRT, GCP, ITN, PLD, PLE, PLF, PLS, PPO, TV2, VLE, VLI, VLO
```

Comment:
The "Full Example" section is missing with examples of .items and .things files.

Reply:
I don't find any "Full Example" section in the developer guide. I've always used the UI and haven't had a need for .items & .things files. Must this really be documented? If so, is there a way to generate such example data?

Reply:
There are many users who use textual configuration. That's why we insist on adding these examples. A stub of the section should've been generated when you created the binding skeleton. See https://github.com/openhab/openhab-core/blob/main/tools/archetype/binding/src/main/resources/archetype-resources/README.md at the bottom. You can of course take a peak at the other binding's readmes.

Reply:
Added "Full Example" section to README.md